config/seeds.go

```diff
@@ -0,0 +1,57 @@
+package config
+
+import (
+	"strings"
+
+	"github.com/spf13/viper"
+	"github.com/tendermint/tendermint/config"
+)
+
+// Seed is used to unmarshal the seeds.toml file
+type Seed struct {
+	Name    string `mapstructure:"name"`
```

Comment:
Looks like `name` is mandatory though we don't actually use it.  I'm fine to keep it mandatory if you like.

Reply:
I don't think it is mandatory, I could make it clearer that it's not needed by not even parsing it if you'd like?